pyproject.toml

```diff
@@ -18,7 +18,7 @@ classifiers = [
 	"Programming Language :: Python :: 3.12",
 	"Programming Language :: Python :: 3.13",
 ]
-dependencies = ["attrs<=25.1.0,>=21.2.0", "importlib-metadata<=8.5.0,>=6.0", "python-slugify<=8.0.4,>=5.0.0", "fsspec<=2025.2.0,>=0.4.0"]
```

Comment:
Although we can don't touch the lower borders...

Reply:
Yes, can you revert the change to the lower bounds? For a library like ours we would like to ensure we are as permissive as possible with our dependency requirements

Reply:
Ok.